src/entities/models.ts

```diff
@@ -58,71 +57,48 @@ export interface EntityDefinition<T> {
 }
 
 export interface EntityStateAdapter<T> {
-  addOne<S extends EntityState<T>>(state: S, entity: TypeOrPayloadAction<T>): S
```

Comment:
What's the net benefit of dropping `TypeOrPayloadAction` here?

Reply:
You had the function overload only taking the PayloadAction there anyways, so it's either remove that or cut this back to prevent duplication. 

Since I remembered you mentioned that you had some problem with this `TypeOrPayloadAction` and having two overloads with different arguments somehow looked "cleaner" to me anyways, I decided to go the "two overload" way.